### ISSUE TITLE
How to fine-tune pre-trained model on a smaller dataset?

### ISSUE THREAD

**Issue by alsbhn** [abisee/pointer-generator#162]
I was wondering how it is possible to fine-tune the pre-trained model on a smaller dataset? What about the implementation of coverage mechanism during the fine-tuning? Do you propose specific settings for hyperparameters (learning rate for example) and the number of iterations?
